demos/interactive_face_detection_demo/cpp/interactive_face_detection.hpp

```diff
@@ -13,47 +13,63 @@
 DEFINE_INPUT_FLAGS
```

Comment:
Could you merge this header with main.pp

Reply:
Done